sklearn/utils/_testing.py

```diff
@@ -789,7 +793,7 @@ def _convert_container(
             f"sparse_csr_array is only available with scipy>=1.8.0, got {sp_version}"
         )
     elif constructor_name == "sparse_csc":
-        return sp.sparse.csc_matrix(container, dtype=dtype)
+        return csr_container(dtype=dtype)
```

Comment:
This needs to be changed as suggested above.

Reply:
Oh, I'm sorry. Do you mean as follows??

current version
```python
return csr_container(dtype=dtype)
````

modified version(I will add.)
```python
return csc_container(dtype=dtype)
````

Reply:
The current version of source is as follow,
https://github.com/y-vectorfield/scikit-learn/blob/1d45c34d1f2706fbdd5599644f317e4a3a34934f/sklearn/utils/_testing.py#L761C1-L788C60

Reply:
I meant https://github.com/scikit-learn/scikit-learn/pull/27847#discussion_r1405371726 and https://github.com/scikit-learn/scikit-learn/pull/27847#discussion_r1405371760 but for CSC.